src/components/Accordion/Accordion.tsx

```diff
@@ -32,20 +40,28 @@ export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
  *
  * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-accordion--overview)
  */
-const AccordionComponent = ({ className, children, ...props }: AccordionProps) => {
-  const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Accordion');
-  const variantProps = getAccordionVariantProps(props.variant);
+const AccordionComponent = forwardRef<HTMLDivElement, AccordionProps>(
+  ({ className, children, values, onValuesChanged, ...props }: AccordionProps, ref) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Accordion');
+    const variantProps = getAccordionVariantProps(props.variant);
+
+    return (
+      <Accordion.Root
+        className={classnames(`${baseClassName}`, className)}
+        {...commonProps}
+        {...variantProps}
+        id={props?.id}
+        ref={ref}
+        type="multiple"
```

Comment:
few problems here, `variantProps` is returning `collapsible: true` which is blowing up the domconsole
```
Warning: Received `true` for a non-boolean attribute `collapsible`.

If you want to write it to the DOM, pass a string instead: collapsible="true" or collapsible={value.toString()}.
    at div
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-6OGP3IOB.js?v=e5fade98:276:13
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-6OGP3IOB.js?v=e5fade98:130:11
```

Also, we can't hardcode the type to `multiple`, we have to use the type returned by `{...variantProps}`